### DISCUSSION TITLE
There will be a crash problem after performing evolution.

### DISCUSSION THREAD

**Discussion by Minxiangliu** [ultralytics/yolov5#8778] [Q&A] [answered]
I have read the[ introduction of Hyperparameter Evolution](https://github.com/ultralytics/yolov5/issues/607), and it is successfully executed, but after several evolutions, there will be a crash problem, what should I do?
The timing of each crash is not necessarily.

Command:
`python train.py --img 640 --batch 32 --epochs 20 --workers 4 --data datasets.yaml --weights yolov5s.pt --cache --evolve 500`

![image](https://user-images.githubusercontent.com/30144428/181725871-dc02c7e7-b6a7-458d-8328-16dfb9f04416.png)

![image](https://user-images.githubusercontent.com/30144428/181725838-e11c6ed2-5005-4c9b-a578-4b97d0e4aa6a.png)

**Comment by glenn-jocher** [comment]
@Minxiangliu 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by Minxiangliu** [comment, marked as answer]
Finally I can train normally after removing `--cache` in the command.
